newsfragments/10524.feature.rst

```diff
@@ -1 +1 @@
-Display login and logout button in the homepage
+Display login and logout buttons in the homepage
```

Comment:
This newsfragment should be removed. It's linked to Parsec Account, which is not enabled yet.